images/index.html

```diff
@@ -0,0 +1,269 @@
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button type="button" class="the_btn">See More</button>
+                </div>
+            </div>
+
+
+            <div class="part_1">
+                <div class="card">
+                    <div class="details">
+                        <h2 class="the_title">Multi-Post Stories</h2>
+                        <ul class="item_info">
+                            <li class="alignment1">Facebook</li>
+                            <li class="alignment1">Full Stack Dev</li>
+                            <li class="alignment1">2015</li>
+                        </ul>
+                        <p class="the_content">
+                            <p>Experimental content creation feature that allows users to</p>
+                            <p>add to an existing story over the course of a day without</p>
+                            <p>spamming their friends</p>
+
+                        </p>
+                        <ul class="coding">
+                            <li>Html</li>
+                            <li>Ruby on rails</li>
+                            <li>Css</li>
+                            <li>JavaScript</li>
+                        </ul>
+                        <button type="button" class="the_btn">See Project</button>
+                    </div>
+                    <div class="work_statement">
+                        <img src="images/colorfulbasketball.svg" alt="Project 2" width="500" />
+                    </div>
+
+                </div>
+            </div>
+
+
+            <div class="part_1">
+                <div class="card">
+                    <div class="work_statement">
+                        <img src="images/orange ting.svg" alt="Project 3" width="500">
+                    </div>
+                    <div class="details">
+                        <h2 class="the_title">Facebook 360</h2>
+                        <ul class="item_info">
+                            <li class="alignment1">FACEBOOK</li>
+                            <li class="alignment1">FULL STACK DEV</li>
+                            <li class="alignment1">2015</li>
+                        </ul>
+                        <p class="the_content">
+                            Exploring the future of media in Facebook's first virtual Reality app a place to discover and enjoy photo and enjoy 360 photo and videos on Gear VR
+                        </p>
+                        <ul class="coding">
+                            <li>Html</li>
+                            <li>Css</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <button type="button" class="the_btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+
+
+            <div class="part_1">
+                <div class="card">

```

Comment:
[OPTIONAL] 

You have another index.html file inside the images folder. I would say it's redundant, isn't it?

- [ ] Please make sure to remove this index file from your repo.